composer.json

```diff
@@ -12,10 +12,10 @@
         "jean85/pretty-package-versions": "^1.1",
         "nette/robot-loader": "^3.0",
         "nette/utils": "^2.5",
-        "nikic/php-parser": "4.0.x-dev",
+        "nikic/php-parser": "^4.0",
         "phpdocumentor/reflection-docblock": "^4.3",
         "phpdocumentor/type-resolver": "^0.4",
-        "rector/better-reflection": "^3.0.6",
+        "roave/better-reflection": "dev-master#dd2c2d0",
```

Comment:
To really use stable dependencies, here use `rector/better-reflection` and update it first to `nikic/php-parser` 4.0.
I'll release new tag for it then